src/main/scala/chargepoint/docile/test/InteractiveOcppTest.scala

```diff
@@ -49,10 +49,10 @@ object InteractiveOcppTest {
 
     protected def connectionData: OcppConnectionData
 
-    def q: Unit =
+    def q(): Unit =
```

Comment:
...makes the compiler shut up but makes the interactive interface less usable. A better fix for the warning would be to disable it in build.sbt imo.